linetools/spectra/io.py

```diff
@@ -412,18 +412,20 @@ def parse_UVES_popler(hdulist):
     xspec1d = XSpectrum1D.from_tuple((uwave, fx, sig, co))
     return xspec1d
 
-def parse_FITS_binary_table(hdulist, exten=None, flux_tag=None,
-                            sig_tag=None, co_tag=None):
+def parse_FITS_binary_table(hdulist, exten=None, wave_tag=None, flux_tag=None,
+                            sig_tag=None, co_tag=None, var_tag=None):
```

Comment:
Does an ivar_tag also need to be added here? It's listed above as an option in `lt_xspec`.